tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -295,21 +311,13 @@ def test_remove_inactive_service(super_client, admin_client, sim_context, nsp):
     assert instances[0].state == "running"
 
     # deactivate service
-    service = wait_success(admin_client, service.deactivate(), 120)
+    service = wait_success(admin_client, service.deactivate())
     assert service.state == "inactive"
 
     # remove service
-    service = wait_success(admin_client, service.remove(), 120)
+    service = wait_success(admin_client, service.remove())
     assert service.state == "removed"
-    instances = super_client. \
-        list_container(name="compose_" + service.name + "_" + "1")
-    assert len(instances) == 1
-    assert instances[0].state == "removed"
-    instance_service_map = super_client. \
```

Comment:
Not a merge blocker, but was there any particular reason we were checking the state of the mapping before?